deploy/monitor/mdc_prometheus_rule.yaml

```diff
@@ -8,13 +8,41 @@ metadata:
   name: mdc-monitoring
 spec:
   groups:
-    - name: general.rules
-      rules:
-      - alert: MobileDeveloperConsoleDown
-        expr: absent(up{job="example-mdc-mdc"} == 1)
-        for: 5m
-        labels:
-          severity: critical
-        annotations:
-          description: "The Mobile Developer Console has been down for more than 5 minutes."
-          summary: "The Mobile Developer Console is down."
+  - name: general.rules
+    rules:
+    - alert: MobileDeveloperConsoleDown
+      expr: absent(kube_pod_container_status_running{namespace="mobile-developer-console",container="mdc"}>=1)
+      for: 5m
+      labels:
+        severity: critical
+      annotations:
+        description: "The MDC has been down for more than 5 minutes. "
+        summary: "The mobile-developer-console is down. For more information see on the MDC at https://github.com/aerogear/mobile-developer-console"
+        sop_url: "https://github.com/aerogear/mobile-developer-console-operator/blob/0.2.0/SOP/SOP-mdc.adoc"
+    - alert: MobileDeveloperConsoleConsoleDown
```

Comment:
This looks to be a duplicate. I believe we added an extra one in MSS as there was a server and a UI. In the case of MDC I don't believe there is a server so I'm not sure we need to include both of these.

Reply:
It is just because of the name that you have this impression. I did a change in order to address your comment. Please, feel free to check it. I'd prefer we keep the same alert that we have in the other operators. SRE's team requested in order to make it easier we keep a standard for it as well.